sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/PublishEventsWithWebSocketsAndProxy.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs;
+
+
+import com.azure.core.amqp.AmqpTransportType;
+import com.azure.core.amqp.ProxyAuthenticationType;
+import com.azure.core.amqp.ProxyOptions;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Demonstrates how to publish events when using web sockets and a proxy and synchronous producer.
+ */
+public class PublishEventsWithWebSocketsAndProxy {
+    /**
+     * Main method to invoke this sample.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        List<EventData> telemetryEvents = Arrays.asList(
+            new EventData("Roast beef".getBytes(UTF_8)),
+            new EventData("Cheese".getBytes(UTF_8)),
+            new EventData("Tofu".getBytes(UTF_8)),
+            new EventData("Turkey".getBytes(UTF_8)));
+
+        // The connection string value can be obtained by:
```

Comment:
This might be worth putting in README and the samples just provide a link to it.